cloudbuild.yaml

```
@@ -1,38 +1,57 @@
-      - '-t'
-      - '$_AR_HOSTNAME/$PROJECT_ID/cloud-run-source-deploy/$_SERVICE_NAME/$_SERVICE_NAME:$COMMIT_SHA'
-      - '-f'
-      - 'Dockerfile'
-      - .
-
-  - name: gcr.io/cloud-builders/docker
-    args:
-      - push
-      - '$_AR_HOSTNAME/$PROJECT_ID/cloud-run-source-deploy/$_SERVICE_NAME/$_SERVICE_NAME:$COMMIT_SHA'
-
+steps:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)